tekton-example/README.md

```diff
@@ -26,52 +30,84 @@ Creation of a TaskRun (using `promote-run.yaml`) will then further promote from
 
-- To create the PipelineRun, use:
+This example promotes from the `promote-demo` repository, containing a service with the same name.
+
+- This creates a PipelineRun that executes the `service-promote-pipeline`, which will build the code and promote it to a repository you have specified
+- The logs will be outputted to your console, and you can also view its progress in the Tekton Dashboard.
+
+The exact steps to create a PipelineRun depend on whether you are using the v1alpha1 or v1beta1 APIs.
+
+### v1alpha1
+
+To create the PipelineRun using v1alpha1, use:
+
 ```shell
 tkn pipeline start service-promote-pipeline --resource git-source=git-app-repo --resource docker-image=docker-app-image --param commitId=v1 --param github-secret=promote-secret --param commit-name=<yourgitname> --param commit-email=<yourgitemail> --param to=https://github.com/<github username>/<github repo>.git --param service=promote-demo --workspace name=repo-space,claimName=repopvc,subPath=dir -s demo --showlog
 ```
 
-This example promotes from the `promote-demo` repository, containing a service with the same name.
+### v1beta1
 
+One of the main differences between our v1alpha1 and v1beta1 samples is that v1beta1 does not use a Tekton Git PipelineResource. Instead we use Tekton workspaces, which must be backed by persistent storage since they contain more than one task. Persistent storage requires a PersistentVolumeClaim. These can be dynamically generated using the `volumeClaimTemplate` stanza, but this is not supported in `tkn` until https://github.com/tektoncd/cli/issues/1006 is resolved. In the meantime we provide a PipelineRun:
 
-- This creates a PipelineRun that executes the `service-promote-pipeline`, which will build the code and promote it to a repository you have specified
-- The logs will be outputted to your console, and you can also view its progress in the Tekton Dashboard.
+```sh
+kubectl create -f templates/promote-pipelinerun.yaml
+```
+
+You will need to locate and tail the logs of this PipelineRun for yourself. For example you can use, 
+
+```sh
+kubectl get pipelineruns
+tkn pipelinerun logs [pipelinerun] -f
+```
 
 ## Promote to Next Managed Environment
 
 Optionally, you can run a subsequent promote from one GitOps repository to another (e.g staging to prod) after merging the pull request on your first GitOps repository. For this you will need a third repository, and for this you can fork: https://github.com/rhd-gitops-example/gitops-example-staging
 
--  To do this second promote, you will need to create a TaskRun that executes a task promoting from a testing repository to a production repository
+This section has only been implemented under the v1alpha1 sample. We're still working through how promotion between GitOps repositories should work, so this section is fairly expermimental.
```

Comment:
experimental*

Do you think we should call this out earlier? I might have chosen v1alpha1 or v1beta1 early, made some progress, then got stuck here - or at least scared off (I personally don't want to continue)